index.js

```diff
@@ -34,6 +34,6 @@ function isPrivate (ip) {
 }
 
 function toNumber (ip) {
-  const [a, b, c, d] = ip.split('.')
-  return 16777216 * parseInt(a, 10) + 65536 * parseInt(b, 10) + 256 * parseInt(c, 10) + parseInt(d, 10)
+  const [a, b, c, d] = ip.split('.').map(n => parseInt(n, 10))
```

Comment:
mind moving the n => ... function into a static function since it has no state? ie

function toNumber (ip) {
   const ... = ip.split(...).map(toInt)
}

function toInt (n) {
  return parseInt(n, 10)
}

Reply:
It is also possible to use the Number class.

```js
const [a, b, c, d] = ip.split('.').map(Number)
```

Reply:
Ya prefer parseInt so it's less ambigious (no floats etc)